### PR TITLE
Part of #17670 for file beta-header\index.js

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation

In this PR I have worked on the lint issues of the files:

- ui\components\app\beta-header\index.js

### Before
Screenshot (24)

### After
Screenshot (25)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Comment by PrgrmrHarshShukla** [comment]
Hello @brad-decker , @georgewrmarshall 

I worked on the lint issues of the files

- ui/contexts/metametrics.js
- ui/hooks/useNftsCollections.js
- ui/pages/confirm-signature-request/index.js

Please review this PR.

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall , @brad-decker ,

I worked on some more lint issues in the files

- ui\components\app\beta-header\index.js
- ui/contexts/metametrics.js
- ui/hooks/useNftsCollections.js
- ui/pages/confirm-signature-request/index.js

Kindly review the PR.

**Comment by brad-decker** [comment]
@PrgrmrHarshShukla , what "lint issues" are you fixing in these, could you please provide before/after shots of failing lint tests that you resolving?

**Comment by PrgrmrHarshShukla** [comment]
Here I have worked on the lint issues you mentioned in the comments of PR #18609

Screenshot (13)


Major focus was on the last 3 files.

**Comment by PrgrmrHarshShukla** [comment]
Still there are lint issues in which I am finding difficulty to solve.

Screenshot (14)
Screenshot (16)
Screenshot (17)


**I have not been able to properly set it up locally, so, `yarn lint:fix` is not working as expected.**

**Comment by PrgrmrHarshShukla** [comment]
The number of issues shown beside the PROBLEMS tab are all same as shown in the screenshots.

**Comment by PrgrmrHarshShukla** [comment]
**I tried to solve the issues.
I changed the line endings from CRLF to LF.
It solved all of the above mentioned issues.**



Really sorry for the inconvenience in handling the PRs.
I have somewhat messed up the PRs.
Shall I make separate PRs where I work on the issue #17670 "Replace Typography with Text component" and with the lint issues ?

I am using Windows OS.
Screenshot (18)

I am using VSCode.

When I run `yarn`

Screenshot (20)

**Comment by brad-decker** [comment]
@PrgrmrHarshShukla no need to apologize! Using CLRF Is the default on windows machines and you might have to change your editor settings to not conflict with our prettier configuration (which prefers LF, as we are predominantly linux and mac users). 

You may split your prs up as small as you wish to progress #17670 just keep any lint fixes you have alongside those changes. So for example if you have one PR to update Typography to Text in a specific folder of components, make any lint fixes that are needed related to those changes in the same PR. I hope that makes sense. If not let me know and i'll provide follow up guidance. Once you have your PRs organized as such we can then close the "fix lint" specific PRs and keep our feedback together with the intended changes.

Thank you for your diligence and for contributing to MetaMask. We appreciate it

File: ui/pages/confirm-signature-request/index.js [hunk 1]
```diff
@@ -141,6 +148,10 @@ const ConfirmTxScreen = ({ match }) => {
     sendTo,
     unapprovedMessagesTotal,
     unapprovedTxs,
+    dispatch,
+    index,
+    navigateToMostRecentOverviewPage,
+    prevValue,
   ]);
```
Comment (georgewrmarshall) [inline ui/pages/confirm-signature-request/index.js:155]: These files shouldn't be present in this PR

File: ui/hooks/useNftsCollections.js [hunk 2]
```diff
@@ -74,6 +74,8 @@ export function useNftsCollections() {
     prevChainId,
     selectedAddress,
     prevSelectedAddress,
+    previouslyOwnedText,
+    unknownCollectionText,
   ]);
 
   return { nftsLoading, collections, previouslyOwnedCollection };
```
Comment (georgewrmarshall) [inline ui/hooks/useNftsCollections.js:81]: These files shouldn't be present in this PR

File: ui/contexts/metametrics.js [hunk 3]
```diff
@@ -88,7 +88,7 @@ export function MetaMetricsProvider({ children }) {
         options,
       );
     },
-    [context],
+    [context, addContextPropsIntoEventProperties],
   );
 
   // Used to prevent double tracking page calls
```
Comment (georgewrmarshall) [inline ui/contexts/metametrics.js:94]: These files shouldn't be present in this PR

**Review by georgewrmarshall — COMMENTED** [review]
Hey @PrgrmrHarshShukla, looks like you have some unintended file changes in this PR please remove

**Comment by PrgrmrHarshShukla** [comment]
> Hey @PrgrmrHarshShukla, looks like you have some unintended file changes in this PR please remove

Hello @georgewrmarshall 
I have deleted the mentioned files.
Please check.

**Comment by PrgrmrHarshShukla** [comment]
I have also added the before and after screenshots in the description.

@georgewrmarshall 
Please check.

**Review by georgewrmarshall — COMMENTED** [review]
Hey @PrgrmrHarshShukla, we don't want to remove these files we just don't want to change them in any way. This PR should only contain changes to `ui/components/app/beta-header/index.js` file anything else should not be included. Thanks!

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,
Thank you for being so supportive.

I have made the changes as needed.
Please check.

**Review by georgewrmarshall — DISMISSED** [review]
Nicely done! 👍 Excellent work on the before/after screenshots they help a lot! LGTM 💯

**Comment by georgewrmarshall** [comment]
Ah apologies just noticed that snapshot needed to be updated.

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, It looks like https://github.com/MetaMask/metamask-extension/pull/18964 was merged before this and made the changes necessary to complete this task. As we don't assign specific issues for https://github.com/MetaMask/metamask-extension/issues/17670 because it is so large I like to apologize. Thanks again for your contribution and I encourage you to continue contributing!

**Pull request closed without merging**